contribs/gmf/src/search/component.js

```diff
@@ -938,6 +928,12 @@ export class SearchController {
     // based on https://github.com/twitter/typeahead.js/issues/780#issuecomment-251554452
     // FIXME: remove this workaround when https://github.com/corejavascript/typeahead.js/issues/60 is fixed
 
+    //if there is no clear button clean the map and colorpicker
```

Comment:
```suggestion
    // If there is no clear button clean the map and colorpicker.
```
I think that's a good improvement, but it was not asked by any customer. If you merge that, we should present that carefully during the demo.